### PR TITLE
Feature/#42 new bot api version

### LINKED ISSUE #42: Bot api version 5 (November 4, 2020)
Need to update library to support [Bot api version 5 (November 4, 2020)](https://core.telegram.org/bots/api#november-4-2020)

### PR DESCRIPTION
**Pull request by greenplugin**
## Description

November 4, 2020
Bot API 5.0

## Motivation and context

#42 

## How has this been tested?

Unit tests

## Types of changes

What types of changes does your code introduce?:
 - New feature (non-breaking change which adds functionality) 

## Changelist
- tbd

Closes #42

### CODE REVIEW

File: tests/Method/CopyMessageMethodTest.php [hunk 1]
```diff
@@ -20,7 +21,6 @@ public function testCreate(): void
             'parseMode' => HasParseModeVariableInterface::PARSE_MODE_MARKDOWN_V2,
             'disableNotification' => true,
             'replyToMessageId' => 0,
-//                    'allowSendingWithoutReply' => true,
```
Comment (Big-Shark) [inline tests/Method/CopyMessageMethodTest.php]: What's going on?

Comment (greenplugin) [reply tests/Method/CopyMessageMethodTest.php]: whoops, will fix

File: tests/Method/CopyMessageMethodTest.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    use ReplyKeyboardMarkupTrait;
+
+    public function testCreate(): void
+    {
+        $method = CopyMessageMethod::create(1, 2, 3, [
+            'caption' => 'caption',
+            'parseMode' => HasParseModeVariableInterface::PARSE_MODE_MARKDOWN_V2,
+            'disableNotification' => true,
+            'replyToMessageId' => 0,
+//                    'allowSendingWithoutReply' => true,
```
Comment (Big-Shark) [inline tests/Method/CopyMessageMethodTest.php]: Same

File: tests/Method/CopyMessageMethodTest.php (hunk 1 above)
Comment (greenplugin) [reply tests/Method/CopyMessageMethodTest.php]: I checked, It's fixel in the last commits https://github.com/tg-bot-api/bot-api-base/pull/44/files#diff-9140e67e920de792745d93cdf7b1d89e21f4122c206263c6479fa45f961942b8

**Review by Big-Shark — APPROVED** [review]
Good job.

**Comment by greenplugin** [comment]
> Good job.

Thanks, I'll fix builds and merge tomorrow

**Review by Big-Shark — DISMISSED** [review]

**Comment by TN8ru5omyg2kf94j6SH7DyEENPJkFY2ftZ** [comment]
A

**Pull request merged**